README.adoc

```diff
@@ -279,9 +263,14 @@ The Dockerfile is provided at the `start` directory. If you're unfamiliar with D
 check out the https://openliberty.io/guides/containerize.html[Containerizing microservices^] guide,
 which covers Dockerfile in depth.
 
-Navigate to the `start` directory and run the following command:
+Run the `mvn package` command from the `start` directory so that the `.war` file resides in the `target` directory.
+To build and containerize the application, start your Docker daemon and then run the
+following Docker build command in the `start` directory. 
+
+Navigate to the `start` directory and run the following commands:
```

Comment:
I'm a little confused by "To build and containerize the application, start your Docker daemon and then run the following Docker build command in the `start` directory." followed by "Navigate to the `start` directory and run the following commands:" 
I believe that the docker build command we're referring to is included after the maven package goal. Would this be improved by making these two sentence into one that reads:
"To build and containerize the application, start your Docker daemon and then run the following commands in the `start` directory: "